migrations/20220830041349_add_mfa_schema.up.sql

```diff
@@ -43,12 +43,11 @@ alter table auth.sessions add column if not exists aal aal_level null;
 
 -- add factor_id and amr claims to session
 create table if not exists auth.mfa_amr_claims(
-    id uuid not null,
     session_id uuid not null,
     created_at timestamptz not null,
     updated_at timestamptz not null,
-    sign_in_method text not null,
-    constraint mfa_amr_claims_session_id_pkey unique(session_id, id),
+    authentication_method text not null,
+    constraint mfa_amr_claims_session_id_authentication_method_pkey unique(session_id, authentication_method),
```

Comment:
This file still uses the old name, even though we merged the one yesterday with a new date into master. Any changes to the DB schema now has to go through a proper new migration file -- although if you put this out today it may be still doable within the same file.